message.go

```diff
@@ -119,6 +119,24 @@ func encodeHeader(s string) string {
 	return mime.QEncoding.Encode("utf-8", s)
 }
 
+func toLowerTrimmed(s string) string {
+	return strings.TrimSpace(strings.ToLower(s))
```

Comment:
Do we really need to trim spaces here? I haven't encountered fields with spaces in the wild yet.

Reply:
Yah I don't know anything about email; wanted to get your guys' thoughts.  I removed it.